pkgs/by-name/pl/plus-jakarta-sans/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
+stdenvNoCC.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

Nit: Use `finalAttrs` is [a good practice](https://github.com/NixOS/nixpkgs/issues/315337) as it's override friendly